src/tests/ampDefaultEpic.ts

```diff
@@ -29,7 +30,7 @@ export function ampDefaultEpic(geolocation?: string): AMPEpicResponse {
                 cta: {
                     text: 'Support the Guardian',
                     // TODO - get tracking code
-                    baseUrl: 'https://support.theguardian.com/contribute',
+                    baseUrl: `https://support.theguardian.com/contribute?acquisitionData=%7B%22source%22%3A%22GOOGLE_AMP%22%2C%22componentType%22%3A%22ACQUISITIONS_EPIC%22%2C%22componentId%22%3A%22${campaignCode}%22%2C%22campaignCode%22%3A%22${campaignCode}%22%7D&INTCMP=${campaignCode}`,
```

Comment:
Is there anything useful we can put in the componentId section? Should it have the same value as the Epic component on Apple News?

Reply:
Typically it's the same as the campaignCode, I'm not sure what would be a useful value here.
`An ID that can be used to distinguish different instances of a given component type`
https://github.com/guardian/ophan/blob/master/event-model/src/main/thrift/acquisition.thrift#L130